manifests/charts/gateways/istio-ingress/values.yaml

```diff
@@ -9,6 +9,8 @@ _internal_defaults_do_not_set:
       labels:
         app: istio-ingressgateway
         istio: ingressgateway
+        # By default, don't enroll gateways into the ambient dataplane
```

Comment:
So in these charts (not sure about `gateway`, but this one) this ends up in deployment matchLabels. so its immutable. I think we are going to need a new field for podLabels (that aren't in the selector) or something

Reply:
`gateway` separates them into `selectorLabels`, yep

Reply:
Since the `ingress/egress` charts are effectively deprecated anyway, I hardcoded it there to KISS.